integration-tests/tests/run-analysis-azure.ts

```diff
@@ -70,7 +70,7 @@ const testRunAnalysisAzure = _.flowRight(
   await findElement(page, clickable({ textContains: 'Creating' }));
 
   // Wait for env to finish creating, or break early on error
-  await findElement(page, clickable({ textContains: 'Running' }), { timeout: Millis.ofMinutes(20) });
+  await findElement(page, clickable({ textContains: 'Running' }), { timeout: Millis.ofMinutes(25) });
```

Comment:
Why is this one 5 minutes less than the general timeout?

Reply:
the test takes some time in addition to waiting for this, namely another 5min wait for the kernel https://github.com/DataBiosphere/terra-ui/pull/4831/files#diff-2265c6f035bb6bf4135d196f0aef358f81f1242986b4737a6a6987f774a28f73L89